README.md

```diff
@@ -135,7 +135,7 @@ var listener = {
   error: (err) => {
     console.error('The Stream gave me an error: ', err);
   },
-  completed: () => {
+  complete: () => {
```

Comment:
Oops, missed that! I've actually done it in both locations so everything should be fine.